content/tagging/assigning_tags.md

```diff
@@ -13,25 +13,195 @@ further_reading:
+
+{{< img src="tagging/assigning_tags/monitortags.png" alt="Manage Monitors Tags" responsive="true" style="width:80%;">}}
+
+When creating a monitor, assign monitor tags under step 4 *Say what's happening*:
+
+{{< img src="tagging/assigning_tags/monitorindivdualtags.png" alt="Create Monitor Tags" responsive="true" style="width:80%;">}}
+
+[63]: /monitors/manage_monitor/
+
+{{% /tab %}}
+{{% tab "Distribution Metrics" %}}
+
+Assign tag keys within [Distribution Metrics][64] (Beta) to create aggregate timeseries by applying sets of tags to a metric, for which a timeseries is created for every combination of tag values within the set.
+
+**Sets of tags are limited to groups of four**:
+
+{{< img src="tagging/assigning_tags/distributionmetricstags.png" alt="Distribution Metrics Tags" responsive="true" style="width:80%;">}}
+
+[64]: /graphing/metrics/distributions/
+
+{{% /tab %}}
+{{% tab "Integrations" %}}
+
+The [AWS][60] integration tile allows you to assign additional tags to all metrics at the account level. Use a comma separated list of tags in the form `key:value`.
+
+{{< img src="tagging/assigning_tags/integrationtags.png" alt="AWS Tags" responsive="true" style="width:80%;">}}
+
+[60]: /integrations/amazon_web_services/
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### API
+
+{{< tabs >}}
+{{% tab "Assignment" %}}
+
+Tags can be assigned in various ways with the [API][65]. See the list below for links to those sections:
+
+- [Post a check run][66]
+- [Post an event][67]
+- [AWS Integration][68]
+- [Post timeseries point][69]
+- [Create][70] or [Edit][71] a monitor
+- [Add][72] or [Update][73] host tags
+- [Send traces][74]
+
+[65]: /api/
+[66]: /api/?lang=python#post-a-check-run
+[67]: /api/?lang=python#post-an-event
+[68]: /api/?lang=python#aws
+[69]: /api/?lang=python#post-timeseries-points
+[70]: /api/?lang=python#create-a-monitor
+[71]: /api/?lang=python#edit-a-monitor
+[72]: /api/?lang=python#add-tags-to-a-host
+[73]: /api/?lang=python#update-host-tags
+[74]: /api/?lang=python#send-traces
+
+{{% /tab %}}
+{{% tab "Example" %}}
+
+Tagging within Datadog is a powerful way to gather your metrics
+and makes scaling your infrastructure a breeze.
+
+For a quick example to demonstrate the power of tagging, perhaps you're
+looking for a sum of two metrics, which you might normally define as follows:
+
+```
+Web server 1: api.metric('page.views', [(1317652676, 100), ...], host="example.com")
+Web server 2: api.metric('page.views', [(1317652676, 500), ...], host="example.com")
+```
+
+What we recommend doing is leaving off the hostname; it then defaults to the host that is sending that point, since they're different hosts it's treated as different points:
```

Comment:
The example is wrong, since we shouldn't set the same host tag for two different Web server.

I think we should refactor this example to show that two metrics can be send from two host for the same domain. Allowing to aggregate on the `domain` tag and to split by `host` tag

Reply:
I think the example is showing the wrong way to do it on purpose then correcting it. I will refactor to only show the correct way to do it.